pnp-helper/src/main/java/samples/com/microsoft/azure/sdk/iot/helper/PnpHelper.java

```diff
@@ -0,0 +1,73 @@
+package samples.com.microsoft.azure.sdk.iot.helper;
+
+import com.google.gson.JsonObject;
+import com.microsoft.azure.sdk.iot.service.devicetwin.*;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class PnpHelper {
+    public static Set<Pair> CreatePropertyPatch(String propertyName, String propertyValue)
```

Comment:
I've added doc comments for these public helpers: https://github.com/Azure/azure-iot-sdk-java/blob/abmisr/thermostat/pnp/src/main/java/com/microsoft/azure/sdk/iot/pnphelpers/PnpHelper.java#L66-L76

Reply:
I will anyway have to rebase so this is mostly a place holder.